sources/packages/web/src/services/http/InstitutionLocation.ts

```diff
@@ -12,6 +12,7 @@ import {
   OptionItemAPIOutDTO,
 } from "@/services/http/dto";
 
+import { ScholasticStandingAPIInDTO } from "./dto/ScholasticStanding.dto";
```

Comment:
Please move this import to the above already imported `from "@/services/http/dto"`.